compiler/src/dmd/canthrow.d

```diff
@@ -22,7 +22,6 @@ import dmd.declaration;
 import dmd.dsymbol;
 import dmd.errorsink;
 import dmd.expression;
-import dmd.expressionsem;
```

Comment:
This seems to be the only import that is added in the PR that introduced the regression. You could just put this as a local (potentially, selective) import at line 83.